lms/djangoapps/discussion/rest_api/tasks.py

```diff
@@ -28,3 +28,21 @@ def send_thread_created_notification(thread_id, course_key_str, user_id):
     course = get_course_with_access(user, 'load', course_key, check_if_enrolled=True)
     notification_sender = DiscussionNotificationSender(thread, course, user)
     notification_sender.send_new_thread_created_notification()
+
+
+@shared_task
```

Comment:
do we need this to be a task? it would have a small audience.

Reply:
Ideally it should be a task. If it is not a task, it will only increase API response time. (Our API's are already very slow)